packages/webapp/tailwind.config.js

```diff
@@ -30,6 +31,9 @@ module.exports = {
                 'dark-700': '#18181B',
                 'dark-800': '#09090B',
                 'bg-dark-blue': '#182633',
+                'state-green-900': '#84D65A',
```

Comment:
what does state green mean and why is 400 the same as 900?

Reply:
lack of better word, it's used for the state (status) of logs.
good catch on 400 thx